apps/richie/templates/app/bc.yml.j2

```diff
@@ -29,7 +29,12 @@ spec:
   source:
     dockerfile: |-
       FROM {{ richie_image_name }}:{{ richie_image_tag }}
-      # Add new statements here
+      USER 0
+      {% if is_apm_enabled -%}
```

Comment:
add a comment line because this Dockerfile extract will receive several separate augmentations?